panacea/query_client.go

```diff
@@ -13,13 +14,16 @@ import (
 	aoltypes "github.com/medibloc/panacea-core/v2/x/aol/types"
 	"github.com/medibloc/panacea-doracle/config"
 	sgxdb "github.com/medibloc/panacea-doracle/store/sgxleveldb"
-	"github.com/tendermint/tendermint/libs/log"
+	log "github.com/sirupsen/logrus"
+	tmlog "github.com/tendermint/tendermint/libs/log"
+	"github.com/tendermint/tendermint/libs/sync"
```

Comment:
I recommend you to use the `"sync"` package provided by Go by default.
https://pkg.go.dev/sync
https://go.dev/tour/concurrency/9

Actually, the `"github.com/tendermint/tendermint/libs/sync"` is a wrapper of the `"sync"`, but I think you don't have any particular reason to use the Tendermint one.

Reply:
Ah, I got it. It's my mistake. Thanks! I fixed it in [0a8904b](https://github.com/medibloc/panacea-doracle/pull/21/commits/0a8904ba57ea69fd3fd4694cc5779fa85f47d8f3).